docs/adr/0003-End-to-end-tests.md

```diff
@@ -0,0 +1,57 @@
+# 3. End to end tests
+
+Date: 2023-04-18
+
+## Status
+
+Accepted
+
+## Context
+
+End to end (e2e) tests only run tests against exporters `/metrics` endpoint in OpenShift today (using Prow CI). We want to run them with more cases, like against Grafana, making them more reliable.
+
+## Decision
+
+e2e tests will be more user friendly.
+
+Prow CI will run e2e tests that cover more scenarios and are more reliable.
+
+### How
+
+- We will rewrite the e2e tests script in Python.
+    - easier to write and read it
+    - they will run against Pelorus operator (using test image) and not helm
+
+- We will write better documentation on what e2e tests do (script help and read the docs).
+    - more accessible to new developers of the Pelorus project
+
+- We will log only relevant information on tests run.
+    - Too much log information is not good to identify a simple error
+
+- We will not use `mig-demo-apps` anymore.
+    - make development faster (only one PR and everything in one repository)
+    - do not use yaml files for different tests scenarios
```

Comment:
I do not understand this point.

Reply:
Example: in the mig-demo-apps PRs ([like this one](https://github.com/konveyor/mig-demo-apps/pull/101/files)), we write the almost the same thing 3 times in 3 different files. We should use just programming logic to change the behavior instead of files